test/unit/query/builder.js

```diff
@@ -3238,22 +3238,22 @@ describe('QueryBuilder', () => {
       qb()
         .select('*')
         .from('users')
-        .orderBy(['email', { column: 'age', order: 'desc' }]),
+        .orderBy(['email', { column: 'age', order: 'desc' }, 3]),
```

Comment:
ok for this test, can you add another one with query like with chained orderBy ?:

knex('t').select('a', 'b', 'c').orderBy([1, 2]).orderBy(3).toString();

Reply:
done